.github/workflows/deploy-manual.yml

```diff
@@ -16,9 +16,8 @@ jobs:
     uses: ./.github/workflows/use-deploy-ecs.yml
     with:
       deployment-env: ${{ github.event.inputs.env }}
+      iam-role-arn: arn:aws:iam::434035161053:role/github-actions-role-dotcom
```

Comment:
Instead of hard-coding this, let's make it a [repository variable](https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository).

Reply:
Created `AWS_IAM_ROLE` with this value.